nsd.conf.5.in

```diff
@@ -987,10 +987,52 @@ Feed updated zone to verifier over standard input. Default is
 verifier\-feed\-zone configured in
 .B verify:\fR.
 .TP
-.B verifier\-timeout: <seconds>
+.B verifier\-timeout:\fR <seconds>
 Number of seconds before verifier is forcefully terminated. Specify 0 (zero)
 to not use a specific timeout. Default is verifier\-timeout from
 .B verify:\fR.
+.TP
+.B catalog:\fR <consumer or producer>
+If set to \fIconsumer\fR, catalog zone processing is enabled for the zone.
+Only a single zone may be configured as a catalog consumer zone. When more than
+one catalog consumer zone is configured, none of them will be processed.
+Member zones of the catalog will use the pattern specified by the group
+property, or if a group property is missing or invalid, the pattern specified
+by the \fBcatalog-member-pattern\fR option is used. Group properties are valid
```

Comment:
```suggestion
by the \fBcatalog\-member\-pattern\fR option is used. Group properties are valid
```
Escape dashes in the man page (because it's also escaped elsewhere in the manpage)